modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/BookmarksFragment.kt

```diff
@@ -103,25 +107,54 @@ class BookmarksFragment : BaseFragment() {
                 Surface(modifier = Modifier.nestedScroll(rememberNestedScrollInteropConnection())) {
                     BookmarksPage(
                         playerViewModel = playerViewModel,
+                        bookmarksViewModel = bookmarksViewModel,
                         onRowLongPressed = { bookmark ->
                             multiSelectHelper.defaultLongPress(
                                 multiSelectable = bookmark,
                                 fragmentManager = childFragmentManager,
                                 forceDarkTheme = true,
                             )
                         },
+                        showOptionsDialog = { showOptionsDialog(it) }
                     )
                 }
             }
         }
     }
+
+    private val showOptionsDialog: (Int) -> Unit = { selectedValue ->
+        activity?.supportFragmentManager?.let {
+            OptionsDialog()
+                .setForceDarkTheme(true)
+                .addTextOption(
+                    titleId = LR.string.bookmarks_select_option,
+                    imageId = IR.drawable.ic_selectall,
```

Comment:
```suggestion
                    imageId = IR.drawable.ic_multiselect,
```


The icon doesn't seem to match the design.

| Design | Code |
| --- | --- |
| <img width="481" alt="Screenshot 2023-07-21 at 10 08 03 pm" src="https://github.com/Automattic/pocket-casts-android/assets/308331/41edb79c-0cca-414a-8563-653fb02c1fd4"> | <img width="444" alt="Screenshot 2023-07-21 at 10 09 56 pm" src="https://github.com/Automattic/pocket-casts-android/assets/308331/fbe85421-db9e-49ae-9c56-e51dc891f88a"> | 

I exported this one instead.
ic_multiselect.xml
```
<vector xmlns:android="http://schemas.android.com/apk/res/android"
    android:width="24dp"
    android:height="24dp"
    android:viewportWidth="24"
    android:viewportHeight="24">
  <path
      android:pathData="M17,17V20C17,21.105 16.105,22 15,22H4C2.895,22 2,21.105 2,20V9C2,7.895 2.895,7 4,7H7V4C7,2.895 7.895,2 9,2H20C21.105,2 22,2.895 22,4V15C22,16.105 21.105,17 20,17H17ZM20,4H9V7H15C16.105,7 17,7.895 17,9V15H20V4ZM13.665,11.253C14.078,11.621 14.114,12.253 13.747,12.665L8.615,18.426L5.321,15.381C4.916,15.007 4.891,14.374 5.266,13.968C5.641,13.563 6.273,13.538 6.679,13.913L8.476,15.574L12.253,11.335C12.621,10.922 13.253,10.886 13.665,11.253Z"
      android:fillColor="#ffffff"
      android:fillType="evenOdd"/>
</vector>
```